low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -247,12 +253,18 @@ def process(self):
             Conf.set(SSPL_CONFIG_INDEX,
                      "NODEDATAMSGHANDLER>high_memory_usage_wait_threshold",
                      memory_usage_alert_wait)
+            Conf.set(SSPL_CONFIG_INDEX, "IEMSENSOR>log_file_path",
+                     iem_log_file)
             Conf.save(SSPL_CONFIG_INDEX)
             shutil.copyfile(sspl_test_backup, sspl_test_file_path)
             if rc != 0:
                 raise TestException(
                     "%s - ERROR: %s - CMD %s" % (self.name, error, CMD))
 
+            # Remove IEM test specific file
+            if os.path.exists(self.test_iem_file):
+                os.remove(self.test_iem_file)
+
```

Comment:
why these changes required?
instead, sspl will soon disable the iem sensor

Reply:
`why these changes required?`
IEM sensor used to read iem_message file. Before test starts, a dummy file will be mapped to IEM.
During IEM sensor running time, test removes it to fail IEM sensor purposefully.
Then recreates the file and checks sensor recovery.

`instead, sspl will soon disable the iem sensor`
Then we need to go for NodeHwSesnor. I will change this in test. Thanks for letting me know.

Reply:
Updated the test to use NodeHWsensor instead of IEMsensor. Please review @gargpalak048